app/layout/scenarios/edit/analysis/adjust-planning-units/component.tsx

```diff
@@ -71,6 +114,10 @@ export const ScenariosSidebarAnalysisSections: React.FC<ScenariosSidebarAnalysis
     const { includedDefault, excludedDefault } = PUData;
     setClearing(true);
 
+    // Clear all temp PU values
+    dispatch(setTmpPuIncludedValue(includedDefault));
```

Comment:
This is what makes the `Clear` button also clear ongoing changes from the map, as to not confuse the user.